run

```diff
@@ -10,6 +10,10 @@ else
   rm /tmp/server-*.pem
 fi
 
+# This is a hack due to musl libc's DNS handling
+cat /etc/resolv.conf | head -n 3 > dns
+cat dns > /etc/resolv.conf
```

Comment:
Right, but there could be no comment or more than 2 lines comment in the file, so we can do it like this.